tests/test_config_zones.py

```diff
@@ -0,0 +1,163 @@
+import json
+import unittest
+from pathlib import Path
+
+from deepdiff import DeepDiff
+
+from electricitymap.contrib.config import (
+    generate_all_neighbours,
+    generate_zone_neighbours,
+    zone_bounding_boxes,
+    zone_parents,
+)
+
+CONFIG_DIR = Path(__file__).parent.parent.joinpath("config").resolve()
+
+
+class TestConfigZones(unittest.TestCase):
+    def test_bounding_boxes_basic(self):
+        zones = {
+            "AD": {
+                "bounding_box": [[0.906, 41.928], [2.265, 43.149]],
+            },
+            "XX": {},
+        }
+        self.assertDictEqual(
+            zone_bounding_boxes(zones),
```

Comment:
zones here need to be cast to the correct type if we want to avoid type errors. (Currently `dict[str, unknown]` and it needs to be `Dict[ZonKey, Any]`)

Reply:
Added type casting in this file.